sites/all/themes/orwell/sass/bootstrap/_mixins.scss

```diff
@@ -218,3 +218,14 @@
     }
   }
 }
+
+// Class for quoting an element with the proper quotes.
+@mixin quoted() {
+  &::before {
+    content: '\201C';
+  }
+
+  &::after {
+    content: '\201D';
```

Comment:
\201D and above \201C should be named variables, defined whereever you keep the sass variables

i have no way of knowing what 201C/201D is ;)

Reply:
I'll fix it in the next PR, I've been messing much around with styling in the meantime.

Reply:
Ok.